lib/ember-build.js

```diff
@@ -59,7 +66,11 @@ EmberBuild.prototype._concatenateES6Modules = concatenateES6Modules;
 EmberBuild.prototype._buildTemplateCompilerTree = buildTemplateCompilerTree;
 
 EmberBuild.prototype._init = function init() {
-  this._packages = null;
+  this._packages      = null;
+  this._name          = null;
+  this._namespace     = null;
+  this._skipRuntime   = null;
+  this._skipTemplates = null;
```

Comment:
@twokul @rwjblue are `skipTemplates` and `skipRuntime` a good idea to pass in as options?

Should they live in [`build-config.js`](https://github.com/emberjs/emberjs-build/blob/f2683eaf37dc5128b228f3fd3d8ff2c38a074963/lib/build-config.js)?